asr_msgs/AsrObject[] objects
---
asr_msgs/AsrAttributedPointCloud point_cloud
string output
 
